api/index.js

```diff
@@ -12,56 +12,46 @@ var errors = require('./lib/errors');
 var serverLib = require('./lib/server-lib');
 var remote = require('./lib/remote');
 
-module.exports = {
-  info: {
-    serverStatus: Info.serverStatus,
-    isConnected: Info.isConnected,
-    fee: Info.fee,
-    uuid: Info.uuid
-  },
+function RippleAPI() {
+  this.remote = remote;
+}
```

Comment:
> The way you have it written won't work because each instance will have it's own remote object in the next pull request. 

I don't see how this is a problem?